edgexfoundry-holding/device-onvif-camera
# docs: add operationId, and do not remove schemas for Get commands

Fixes #360 







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions


- Load this version of swager yaml into https://editor.swagger.io/
- verify no schema errors
- verify the schemas exist for GET commands



## New Dependency Instructions (If applicable)
